commons/commons-model-extended-mapping/src/main/kotlin/ebi/ac/uk/extended/mapping/to/ToFile.kt

```diff
@@ -4,12 +4,17 @@ import ebi.ac.uk.extended.model.ExtFile
 import ebi.ac.uk.extended.model.FireFile
 import ebi.ac.uk.extended.model.NfsFile
 import ebi.ac.uk.io.ext.size
+import ebi.ac.uk.model.Attribute
 import ebi.ac.uk.model.BioFile
+import ebi.ac.uk.model.constants.FileFields.MD5
 
 internal const val TO_FILE_EXTENSIONS = "ebi.ac.uk.extended.mapping.to.ToFileKt"
 
 fun ExtFile.toFile(): BioFile =
     when (this) {
-        is NfsFile -> BioFile(filePath, file.size(), type.value, attributes.mapTo(mutableListOf()) { it.toAttribute() })
-        is FireFile -> BioFile(filePath, size, type.value, attributes.mapTo(mutableListOf()) { it.toAttribute() })
+        is NfsFile -> BioFile(filePath, file.size(), type.value, toAttributes())
+        is FireFile -> BioFile(filePath, size, type.value, toAttributes())
     }
+
+private fun ExtFile.toAttributes() =
```

Comment:
I think name `toAttributes` and function is misleading (is an extension function of file)  as we are not transforming a file into attributes, we are transforming file attributes into ext attributes. To make more clear imagine you have a class `human`  that have a `born_country` attribute and one declarate an extension `Human.getPopulation` which obtain the population of the country. 

Even function implementation can be correct, is not correct to model a human with shuch operation.